ckanext/datatablesview/blueprint.py

```diff
@@ -206,6 +206,7 @@ def filtered_download(resource_view_id):
                 u'filters': json.dumps(filters),
                 u'format': request.form[u'format'],
                 u'fields': u','.join(cols),
+                u'bom': True,
```

Comment:
If considering this for upstream use a new configuration option here instead of hard-coding it.